doc/api/buffer.markdown

```diff
@@ -810,6 +987,15 @@ buffer.
     var b = new Buffer(50);
     b.fill("h");
 
+### buffer.address()
+
+Returns a String representation of the hex address of the pointer in memory.
```

Comment:
I feel like including the address on `inspect()` isn't useful for the> 95% of use cases, and since there's now the `address()` function it can be retrieved at any time.